lua/telescope/pickers.lua

```diff
@@ -509,22 +540,30 @@ function Picker:find()
   elseif self.initial_mode ~= "normal" then
     error("Invalid setting for initial_mode: " .. self.initial_mode)
   end
+
+  if self.window.externalize then
+    a.nvim_win_set_config(results_win, {external = true, width=width, height=height})
```

Comment:
probably add space to these lines for consistency. `height=height` -> `height = height`